test/test_make_uninstall.txt

```diff
@@ -0,0 +1,11 @@
+.
```

Comment:
how does this work? why do we add '.' entry here?

Reply:
This file is an output of the `find | sort` command. `.` stands for the current directory.